packages/flutter_test/lib/src/matchers.dart

```diff
@@ -292,7 +292,8 @@ Matcher offsetMoreOrLessEquals(Offset value, { double epsilon = precisionErrorTo
   return _IsWithinDistance<Offset>(_offsetDistance, value, epsilon);
 }
 
-/// Asserts that two [String]s are equal after normalizing likely hash codes.
+/// Asserts that two [String]s or `Iterable<String>`s are equal after
+/// normalizing likely hash codes.
 ///
 /// A `#` followed by 5 hexadecimal digits is assumed to be a short hash code
 /// and is normalized to `#00000`.
```

Comment:
Since `value` is now typed `Object`, maybe explicitly state in this doc that `String` or `Iterable<String>` are acceptable Types for `value`. I know, it's kinda implied in the first doc line...

Reply:
(union types would be nice here...)

Reply:
Done